#
# Small compile fixes (For more involved fixes, please use a separate patch).
#
# Please add the errors from gcc before the diffs to save others having
# to do a compile to figure out what your diff is fixing. Thanks.
#

CONFIG_MTD_NAND_FSL_UPM can be built modular, but needs to
use fsl_lbc_lock, which isn't currently exported.

Signed-off-by: Dave Jones <davej@redhat.com>

diff --git a/arch/powerpc/sysdev/fsl_lbc.c b/arch/powerpc/sysdev/fsl_lbc.c
index 422c8fa..1c6c522 100644
--- a/arch/powerpc/sysdev/fsl_lbc.c
+++ b/arch/powerpc/sysdev/fsl_lbc.c
@@ -16,6 +16,7 @@
 #include <asm/fsl_lbc.h>
 
 spinlock_t fsl_lbc_lock = __SPIN_LOCK_UNLOCKED(fsl_lbc_lock);
+EXPORT_SYMBOL(fsl_lbc_lock);
 
 struct fsl_lbc_regs __iomem *fsl_lbc_regs;
 EXPORT_SYMBOL(fsl_lbc_regs);
